src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java

```diff
@@ -119,7 +119,7 @@ public TestRunnerSpawn createTestRunnerSpawn(
       // for this.
       executionInfo.put(ExecutionRequirements.NO_CACHE, "");
     }
-    executionInfo.put(ExecutionRequirements.TIMEOUT, "" + getTimeout(action).toSeconds());
+    executionInfo.put("timeout", "" + getTimeout(action).toSeconds());
```

Comment:
Why change this to a string literal?

Also, it's unclear to me how `tags` interacts with the `timeout` attribute for test targets (or the `--test_timeout` flag, which overrides it). Can you add integration tests to clarify the behavior?